### PR TITLE
Support new Wazuh version 4.9.2 in wazuh-documentation repository

### PR DESCRIPTION
**Pull request by Rebits**
## Description
This PR adds support for version 4.9.2.

### Docs building
- [X] Compiles without warnings.
### Code formatting and web optimization
- [ ] Uses three spaces indentation.
- [ ] Adds or updates meta descriptions accordingly.
- [X] Updates the `redirects.js` script if necessary (check [this guide](https://github.com/wazuh/wazuh-documentation/blob/master/NEW_RELEASE.md)).
### Writing style
- [ ] Uses present tense, active voice, and semi-formal registry.
- [ ] Uses short, simple sentences.
- [ ] Uses **bold** for user interface elements, _italics_ for key terms or emphasis, and `code` font for Bash commands, file names, REST paths, and code.

### CODE REVIEW

**Comment by Rebits** [comment]
```
➜  wazuh-documentation git:(enhancement/7740-support-4.9.2) make html
sphinx-build -b html -d build/doctrees   source build/html
Running Sphinx v7.0.1
/home/rebits/.local/lib/python3.10/site-packages/requests/__init__.py:102: RequestsDependencyWarning: urllib3 (1.26.20) or chardet (5.2.0)/charset_normalizer (2.0.12) doesn't match a supported version!
  warnings.warn("urllib3 ({}) or chardet ({})/charset_normalizer ({}) doesn't match a supported "
loading translations [en-US]... not available for built-in messages
loading pickled environment... done
building [mo]: targets for 0 po files that are out of date
writing output... 
building [html]: targets for 0 source files that are out of date
updating environment: 0 added, 0 changed, 0 removed
reading sources... 
Copying static files for wazuh-doc-images...[100%] img/loading.gif                                                                                                                           
looking for now-outdated files... none found
no targets are out of date.
build succeeded.

The HTML pages are in build/html.

Build finished. The HTML pages are in build/html.

```

File: source/release-notes/index-4x.rst
Comment (javimed) [inline source/release-notes/index-4x.rst:1]: An entry in the TOC tree ant the bottom of this file is missing for the 4.9.2 release notes document

File: source/release-notes/release-4-9-2.rst [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+Changelogs
+----------
+
+The repository changelogs provide more details about the changes.
+
+Product repositories
+^^^^^^^^^^^^^^^^^^^^
+
+-  `wazuh/wazuh <https://github.com/wazuh/wazuh/blob/v4.9.2/CHANGELOG.md>`__
+-  `wazuh/wazuh-dashboard-plugins <https://github.com/wazuh/wazuh-dashboard-plugins/blob/v4.9.2-2.11.0/CHANGELOG.md>`__
```
Comment (javimed) [inline source/release-notes/release-4-9-2.rst:30]:
```suggestion
-  `wazuh/wazuh-dashboard-plugins <https://github.com/wazuh/wazuh-dashboard-plugins/blob/v4.9.2/CHANGELOG.md>`__
```

**Review by javimed — CHANGES_REQUESTED** [review]

File: source/release-notes/release-4-9-2.rst (hunk 1 above)
Comment (Rebits) [reply source/release-notes/release-4-9-2.rst]: Done in [8c4e18f](https://github.com/wazuh/wazuh-documentation/pull/7742/commits/8c4e18ff557062cad97e924419c1ab6ea61341bf)

File: source/release-notes/index-4x.rst
Comment (Rebits) [reply source/release-notes/index-4x.rst:1]: Done in [d58c0bd](https://github.com/wazuh/wazuh-documentation/pull/7742/commits/d58c0bd7122b24cd1098f3d0f04faf445a6f27fb)

**Review by pro-akim — APPROVED** [review]
LGTM

**Review by javimed — APPROVED** [review]

**Pull request merged**